lib/bundle/cask_dumper.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "cask/cask_loader"
```

Comment:
Would like to avoid including cask code directly here if it can be avoided. Can this not be obtained from the JSON e.g. `brew cask info mongodb --json=v1`?